README.md

```diff
@@ -295,13 +293,13 @@ In this section we will create the IOT Analytics components, analyze data and de
           g. Click Create pipeline
       
 
-Now we have created the IOT Analytics Pipeline lets load the batch data.
+Now we have created the IoT Analytics Pipeline lets load the batch data.
 
 #### Create Container Data Set
 
 A container data set allows you to automatically run your analysis tools and generate results. It brings together a SQL data set as input, a Docker container with your analysis tools and needed library files, input and output variables, and an optional schedule trigger. The input and output variables tell the executable image where to get the data and store the results.
 
-Navigate to AWS IOT Analytics console, in the left navigation pane, choose Analyze.
+Navigate to AWS IoT Analytics console, in the left navigation pane, choose Analyze.
 
      
     1. Click on Analyze -> Data Sets → Create
```

Comment:
We donot have the Analyze options anymore , can you please update it to -> 
Click on Data sets ->  Create

The same changes are required for Datastore, pipeline, channel etc.

Reply:
Hi Neel, this should already be done - can you look at commit 6f7dfc8 which is the latest.

Reply:
ok looks good then, feel free to merge with the main branch. Thanks for your help.